### ISSUE TITLE
`imageAspectRatio: 1:1` doesn't change `width` and `height` of the frame unless they're set explicitly

### ISSUE THREAD

**Issue by dalechyn** [wevm/frog-archived#463]
### Describe the bug

![Screenshot 2024-08-06 at 22 53 31](https://github.com/user-attachments/assets/3ec4ae42-a70b-460e-89e8-b5aeb00d0c00)

### Frog Version

latest

### Check existing issues

- [X] I checked there isn't [already an issue](https://github.com/wevm/frog/issues) for the bug I encountered.

**Comment by drparadox05** [comment]
Hey, can I work on this?

**Comment by dalechyn** [comment]
> Hey, can I work on this?

sure!

**Comment by drparadox05** [comment]
Hey, how can I reproduce this?

**Comment by dalechyn** [comment]
> Hey, how can I reproduce this?

Simply change the aspect ratio to `1:1`!

What would be expected is if aspect ratio is `1:1` to have library automatically to fall back to a `height: 1000` and `width: 1000`.

Should also think about the `frog/ui` which also accepts `height` and `width` properties and don't adjust if aspect ratio is `1:1`. Possibly need to pass aspect ratio as a `createSystem` parameter.

**Comment by dalechyn** [comment]
I spent some time thinking how to pull this off properly.

Since `imageAspectRatio` doesn't do much but sets the `fc:frame:image:aspect_ratio` tag, it's not included as a request to the route that generates an image.

Unless Image Handler is used, the default behavior compresses the image in search parameters and we've seen many issues due to browser's limitations of maximum URL length.
I'd not be willing to add another parameter there since there's a risk to increase URL length and potentially break frames that exist now.

Also, `imageAspectRatio` would then have to be passed to `createSystem`, which made to be unaware of the `Frog` instance itself, and the change would make it aware of it, or would require passing `imageAspectRatio` there explicitly again.

Too many effort for such a little fix.
Closing for now, but if anyone has ideas on how to fix it without breaking the API, you're welcome to!
